docs/pages/management/guides/gcp-tags.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: GCP Tags and Labels as Teleport Agent Labels
+description: How to set up Teleport agent labels based on GCP tags and labels
+h1: Sync GCP Tags/Labels and Teleport agent labels
+---
+
+When running on an Google Compute Engine instance, Teleport will automatically detect and import GCP
+[tags](https://cloud.google.com/resource-manager/docs/tags/tags-overview) (key-value pairs that are
+their own resource) and [labels](https://cloud.google.com/compute/docs/labeling-resources) (key-value
+pairs that are specific to each instance)
+as Teleport labels for SSH nodes, applications, databases, and Kubernetes clusters. Both tags and labels imported
+this way will have the `gcp/` prefix; additionally, tags will receive the `tag/` infix and labels will receive
+the `label/` infix. For example, an instance with label `foo=bar` and tag `baz=quux` will have the Teleport labels
+`gcp/label/foo=bar` and `gcp/tag/baz=quux`.
+
+When the Teleport process starts, it fetches all tags and labels from 
+the GCP API and adds them as labels. The process will update the tags every hour, 
+so newly created or deleted tags will be reflected in the labels.
+
+If the label `TeleportHostname` (case-sensitive) is present, its value will override the node's hostname.
```

Comment:
This applies to tag or label? Right?

Reply:
No, just GCP labels. GCP tags are generally meant to be shared by multiple resources, and we don't want to share hostnames between nodes.